robocop/checkers/lengths.py

```diff
@@ -196,7 +196,7 @@ class NumberOfReturnedArgsChecker(VisitorChecker):
             "number-of-returned-values",
             "Too many return values (%d/%d)",
             RuleSeverity.WARNING,
-            ("max_returns", "max_returns", int)
+            ("max_returns", "max_returns", int, 'allowed number of returned values from keyword')
```

Comment:
Split into multiple lines

Reply:
Also what about other rules? 😃

Reply:
It was just for testing :D I can go through all rules with configurables and update the meta info (its optional BTW)

Reply:
I would add it in the places where it's not obvious. Or maybe everywhere? Not sure...

Reply:
Or only where there is enumerated value